misc/dist/macos_tools.app/Contents/Info.plist

```diff
@@ -38,10 +40,15 @@
 	<string>NSApplication</string>
 	<key>LSApplicationCategoryType</key>
 	<string>public.app-category.developer-tools</string>
-	<key>LSMinimumSystemVersion</key>
-	<string>10.12</string>
+	<key>LSArchitecturePriority</key>
+	<array>
+		<string>arm64</string>
+		<string>x86_64</string>
+	</array>
 	<key>LSMinimumSystemVersionByArchitecture</key>
 	<dict>
+		<key>arm64</key>
+		<string>13.5</string>
```

Comment:
That's definitely not an acceptable change, Metal renderer implementation might require 13.5, but editor in general should have as lower min version as possible (11.0 for arm64 and 10.12 for x86).

Reply:
Given Metal is only supported on arm64, macOS 11 should hopefully be ok. Will update the PR

Reply:
PR updated back to macOS 11.0